doc-Managing_Infrastructure_and_Inventory/topics/Networks.adoc

```diff
@@ -0,0 +1,18 @@
+[[cloud-networks]]
+
+== Creating and Administering Cloud Networks
+
+Create a network to provide instances a place to communicate internally and receive IP addresses using Dynamic Host Configuration Protocol (DHCP). A network can also be integrated with external networks in your Red Hat OpenStack Platform deployment or elsewhere, such as the physical network.  
+
+
+[NOTE]
+====
+* Keystone API v3 is required to create cloud tenants on Red Hat OpenStack Platform cloud providers. For more information, see https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/11/html-single/architecture_guide/#comp-identity[OpenStack Identity (keystone)] in the Red Hat OpenStack Platform _Architecture Guide_.
+* Please see the link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/12/html-single/networking_guide/#add_an_interface[_Networking Guide_] in the Red Hat OpenStack Platform documentation for further details on cloud network objects and administration. 
```

Comment:
It's okay to start without please when directing users to other documentation: See the Networking Guide..